etl/steps/data/garden/chartbook/2024-03-21/altimir_1986.py

```diff
@@ -0,0 +1,43 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("altimir_1986")
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["altimir_1986"].reset_index()
+
+    #
+    # Process data.
+    #
+    tb = geo.harmonize_countries(
+        df=tb,
+        countries_file=paths.country_mapping_path,
+    )
+
+    # Remove observations for year 1972 (Altimir does not trust in Gas del Estado's methodology)
+    tb = tb[tb["year"] != 1972].reset_index(drop=True)
```

Comment:
```suggestion
    tb = tb.loc[tb["year"] != 1972].reset_index(drop=True)
```

Apparently, this is the recommended way to access data by index in pandas for production code [[ref](https://pandas.pydata.org/docs/user_guide/indexing.html)]